### ISSUE TITLE
[Request]: Clip Studio Paint EX

### ISSUE THREAD

**Issue by Hexality** [bottlesdevs/Bottles#3383]
### Tell us the problem or your need

I need to install CSPEX 3.0, but I get `PackTouchHitTestingProximityEvaluation` error when I try to launch it after installing.

### Describe the solution you'd like

Pre-configured installing of Clip Studio Paint EX on "Install Programs" section.

**Comment by xioren** [comment]
That is not much information and sounds like an issue with WINE compatibility not Bottles. You might try different runners if you have not already.

**Comment by Hexality** [comment]
> That is not much information and sounds like an issue with WINE compatibility not Bottles. You might try different runners if you have not already.

Yeah maybe, I was reading microsoft docs and it's related to touchscreen handling of Windows 8 and newer NT systems.
Do you have any ideas on how to report this to them? Their website "issue reporting" part is a bit too confusing...

**Comment by xioren** [comment]
No I don't, sorry.
